client/helpers/get-locations.js

```diff
@@ -1,9 +1,21 @@
-export default function getLocations (project, establishment) {
-  const establishments = (project.establishments || []).filter(e => e.name || e['establishment-name']).map(e => e.name || e['establishment-name']);
+import uniq from 'lodash/uniq';
+
+export default function getLocations (project, establishment, establishments) {
+  let est = establishment;
+  if (project.transferToEstablishment) {
+    est = establishments.find(e => e.id === project.transferToEstablishment);
```

Comment:
I'm a little concerned that this may cause issues if an admin is editing a PPL transfer initiated by the PPL holder, since if I understand it correctly, `establishments` in this context is the current user's establishment list.